Rename additional_login_params to additional_login_parameters

Nnachevvv opened the pull request:
This PR fixes #16472:
The documentation for `auth_settings` is outdated, because it is using `additional_login_params` instead of `additional_login_parameters`. Therefore i have renamed all of the occurrences of `additional_login_params` to `additional_login_parameters`

```
ok  	github.com/hashicorp/terraform-provider-azurerm/internal/services/web	3.778s
```

katbyte reviewed (changes requested):
Thanks for the PR @Nnachevvv - however we can't just change a property as it would be a breaking change.

So what we'll need to do is add a new property with the updated name, and deprecate the old property, make them conflict, as well put that old property behind the 4.0 beta flag (you can search for the 3.0 flag to see how to do this)

@jackofallops - thoughts on this property rename?

Nnachevvv commented:
Hi @katbyte,
Thank you for the clarification. 
I found that the most of the resources, where I had introduced`additional_login_parameters` as parameter are deprecated and will be removed in 4.0. 
However, I fixed the documentation for resources where  `additional_login_parameters` is used instead of `additional_login_params` 
https://github.com/hashicorp/terraform-provider-azurerm/blob/7fc6cb3b5db3a116f556924e204f2178c7d937a8/internal/services/appservice/helpers/shared_schema.go#L389

jackofallops commented on website/docs/d/linux_function_app.html.markdown:
```diff
@@ -141,7 +141,7 @@ An `auth_settings` block exports the following:
 * `active_directory` - An `active_directory` block as defined above.
 
-* `additional_login_params` -A map of Login Parameters sent to the OpenID Connect authorization endpoint when a user logs in.
+* `additional_login_parameters` -A map of Login Parameters sent to the OpenID Connect authorization endpoint when a user logs in.
```
```suggestion
* `additional_login_parameters` - A map of Login Parameters sent to the OpenID Connect authorization endpoint when a user logs in.
```

jackofallops reviewed (approved):
Thanks @Nnachevvv - One minor typo (which I'll push), but otherwise LGTM 👍
